.github/workflows/release.yml

```diff
@@ -79,8 +79,12 @@ jobs:
 
   build-linux:
     name: Build Linux
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
```

Comment:
aligned the platform with CI

Reply:
Is there any possibility of aligning it in the other direction: stick to 20.04, or some other known, stable version?

Reply:
changed every `ubuntu-latest` to `ubuntu-22.04`